src/jdk.jdeps/share/classes/com/sun/tools/jdeps/DependencyFinder.java

```diff
@@ -173,9 +173,9 @@ private Optional<FutureTask<Set<Location>>> parse(Archive archive, Finder finder
         trace("parsing %s %s%n", archive.getName(), archive.getPathName());
         FutureTask<Set<Location>> task = new FutureTask<>(() -> {
             Set<Location> targets = new HashSet<>();
-            for (var cf : archive.reader().getClassFiles()) {
+            archive.reader().processClassFiles(cf -> {
```

Comment:
I prefer the naming convention with forEach for operation to iterate through rather than have a `process` on a reader. Perhaps named like `forEachClassFile`?

Reply:
Done in the update.